site/content/user-guide/amqp-adapter.md

```diff
@@ -3,28 +3,33 @@ title = "AMQP Adapter"
+* A failure caused by a client-side error (e.g invalid message address, content-type, adapter disabled for tenant, etc).
+* A failure caused by a server-side error (e.g sender queue full, no credit available yet to send the message, etc).
 
-In the case of a server-side error, the adapter responds with a *RELEASED* disposition, which indicates to the client that the message was OK but it cannot be delivered due to a failure beyond the control of the client. In this case, the client should attempt to redeliver the message again.
+For a client-side error, the adapter **rejects** the message and provides a reason why it was rejected. In the case of a server-side error, the adapter **releases** the message, indicating to the client that the message was OK but it cannot be delivered due to a failure beyond the control of the client. In this case, the client should attempt to redeliver the message again.
 
 <!--more-->
 
 ## Device Authentication
 
-By default, all Hono protocol adapters require clients (devices or gateway components) to authenticate during connection establishment. This is the preferred way for devices to publish data via protocol adapters. The AMQP adapter can be configured to *disallow* authentication for devices by setting the value of `HONO_AMQP_AUTHENTICATION_REQUIRED` to `false`. In this guide, we will give examples for publishing telemetry and events for *authenticated* and *unauthenticated* clients. We will see in the examples later that the main difference between publishing messages for authenticated and unauthenticated devices is in the format of the AMQP 1.0 **address**. For unauthenticated devices, the address should include the tenant and device identifier.
+By default, all Hono protocol adapters require clients (devices or gateway components) to authenticate during connection establishment. This is the preferred way for devices to publish data via protocol adapters. The AMQP adapter can be configured to *disallow* authentication for devices by setting the value of `HONO_AMQP_AUTHENTICATION_REQUIRED` to `false`. In this guide, we will give examples for publishing telemetry and events for *authenticated* and *unauthenticated* clients. We will see in the examples later that the main difference between publishing messages for authenticated and unauthenticated devices is in the format of the  ** message address**. For unauthenticated devices, the message address should include the tenant and device identifiers.
```

Comment:
IMHO we can simply omit the two last sentences here and maybe add a note to the examples for unauthenticated devices ...